src/test/py/bazel/runfiles_test.py

```diff
@@ -54,7 +57,11 @@ def _AssertRunfilesLibraryInBazelToolsRepo(self, family, lang_name):
     self.AssertExitCode(exit_code, 0, stderr)
     bazel_bin = stdout[0]
 
-    exit_code, _, stderr = self.RunBazel(["build", "//foo:runfiles-" + family])
+    exit_code, _, stderr = self.RunBazel([
+        "build",
+        "--verbose_failures",
+        "--experimental_shortened_obj_file_path",
```

Comment:
Why do we need `--experimental_shortened_obj_file_path`? This flag might be removed in the future.

Reply:
Because otherwise the output path for runfiles.cc in the external repository (bazel_tools) is just too long. :(

Reply:
Ah,, thanks! Then I'll ping you to remove this after short obj file path is switched to default.